packages/office-ui-fabric-react/src/components/Persona/Persona.base.tsx

```diff
@@ -119,14 +130,17 @@ export class PersonaBase extends BaseComponent<IPersonaProps, {}> {
     );
 
     return (
-      <div
-        { ...divProps }
-        className={ classNames.root }
-        style={ coinSize ? { height: coinSize, minWidth: coinSize } : undefined }
-      >
-        <PersonaCoin { ...personaCoinProps } />
-        { (!hidePersonaDetails || (size === PersonaSize.size10 || size === PersonaSize.tiny)) && personaDetails }
-      </div>
+      <FocusZone componentRef={ this._focusZone } >
+        <div
+          { ...divProps }
+          className={ classNames.root }
+          style={ coinSize ? { height: coinSize, minWidth: coinSize } : undefined }
+          tabIndex={ 0 }
```

Comment:
@dzearing I couldn't get the focus zone to work without tabIndex. Is that expected?